opt/cruise-control/start.sh

```diff
@@ -6,4 +6,8 @@ cd /opt/cruise-control
 if [ -z "${KAFKA_HEAP_OPTS}" ]; then
   export KAFKA_HEAP_OPTS="-XX:InitialRAMPercentage=75 -XX:MinRAMPercentage=75 -XX:MaxRAMPercentage=75"
 fi
+
+if [ -z "${KAFKA_OPTS}" ]; then
+  export KAFKA_OPTS="-Dsun.net.inetaddr.ttl=60"
```

Comment:
i see cc uses openjdk-11. i think the property changed to `networkaddress.cache.ttl` ?

Reply:
actually sun.net.inetaddr.ttl seems to be still supported 👍 
https://github.com/AdoptOpenJDK/openjdk-jdk15/blob/master/src/java.base/share/classes/sun/net/InetAddressCachePolicy.java#L34-L36

I see cruise-control is using adoptjdk

Reply:
Hey @amuraru! Thanks for bringing up this. After taking a secon loo on this, it seems neither `networkaddress.cache.ttl` nor `networkaddress.cache.negative.ttll`can be set from command line as these are not system, but security properties.

> Since these 2 properties are part of the security policy, they are not set by either the -D option or the System.setProperty() API, instead they are set as security properties.

From [Network Properties](https://cr.openjdk.java.net/~iris/se/11/latestSpec/api/java.base/java/net/doc-files/net-properties.html)

The good news is that apparently `sun.net.inetaddr.ttl` is still supported in OpenJDK11 to set `networkaddress.cache.ttl` from command line in case it is not explicitly set in `java.security` file. At least this is my understanding after reading the relevant part of the source code: https://github.com/AdoptOpenJDK/openjdk-jdk/blob/jdk-11+28/src/java.base/share/classes/sun/net/InetAddressCachePolicy.java#L88-L122.

Another thing I found by reading the src is that this parameter is set to `30` seconds by default if neither the `networkaddress.cache.ttl` nor `sun.net.inetaddr.ttl` is explicitly set.

Although I might be mistaken here as I am far from being a Java expert. 🙂 

If I got it right then this change is probably no even necessary and the PR can be closed.

Reply:
> Another thing I found by reading the src is that this parameter is set to 30 seconds by default if neither the networkaddress.cache.ttl nor sun.net.inetaddr.ttl is explicitly set.

This is only true in case the Security Manager is enabled. (It is disabled by default). So I think it should be fine to set this explicitly.

@amuraru Let me know what you think.